### ISSUE TITLE
Updates of max_logvar and min_logvar parameters

### ISSUE THREAD

**Issue by sufodsia** [jiangsy/mbpo_pytorch#4]
Hi, thanks for this Pytorch version of MBPO.

In **mbpo_pytorch/models/dynamics.py**,  the maximum and minimum log(var) are initalized as follows :
`        self.max_diff_state_logvar = nn.Parameter(torch.ones([1, state_dim]) / 2.)`
`         self.min_diff_state_logvar = nn.Parameter(-torch.ones([1, state_dim]) * 10.)`
`         self.max_reward_logvar = nn.Parameter(torch.ones([1, reward_dim]) / 2.)`
`         self.min_reward_logvar = nn.Parameter(-torch.ones([1, reward_dim]) * 10.)`
        
Are those parameters updated at any time ? 

I couldn't find it, and then it wouldn't make sense to include them in the loss in **mbpo_pytorch/algos/mbrl/mbpo.py** 
`train_model_loss += \
                    0.01 * (torch.sum(self.dynamics.max_diff_state_logvar) + torch.sum(self.dynamics.max_reward_logvar) -
                            torch.sum(self.dynamics.min_diff_state_logvar) - torch.sum(self.dynamics.min_reward_logvar))`

Best

**Comment by jiangsy** [comment]
these variables are trainable in the original MBPO implementation(https://github.com/JannerM/mbpo/blob/28b1e3b1382dcda34b421961286b59f77770d48c/mbpo/models/bnn.py#L178), using the the loss (https://github.com/JannerM/mbpo/blob/28b1e3b1382dcda34b421961286b59f77770d48c/mbpo/models/bnn.py#L192). 
I just separated them into `state_logvar` and `reward_logvar`.

I am not sure why they use such a complicated update but since it's a reimplementation, i choose to keep this.
